<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifPersonnel</class>
 <widget class="QDialog" name="ModifPersonnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineNom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pr√©nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="linePrenom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxType">
          <item>
           <property name="text">
            <string>Banquier type A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Banquier type B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Assureur logement</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Assureur voiture</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Assureur vie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Informaticien</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Divers</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mot de Passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePassword">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="buttonOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAnnuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
